examples/wearable/UIComponents/contents/assistviews/popup.html

```diff
@@ -85,18 +85,17 @@ <h2 class="ui-title">
 			<div class="ui-popup-content">
 				Agree?
```

Comment:
In debugger it shows font-size 32 px for this statement
Guide says about 30px
Please check @font_size_popup_text

Reply:
Ok. I changed font-size to 30px.
Thanks.